packages/otelbin/src/components/monaco-editor/otelCollectorConfigValidation.ts

```diff
@@ -99,6 +102,28 @@ export function validateOtelCollectorConfigurationAndSetMarkers(
 	}
 	if (!totalErrors.jsYamlError) {
 		customValidate(mainItemsData, serviceItemsData, errorMarkers, totalErrors, configData);
+		const serverSideErrorElement = findErrorElement(serverSideValidationPath, parsedYamlConfig);
```

Comment:
**Suggestion:** Add tests for this :)

Reply:
All the functions that used for serverSide markers are already have tests. (used the same functions as for ajv error markers) @bripkens

Reply:
So if that is the case, does that mean that this new code can be deleted and there will be a red test? :)